### DISCUSSION TITLE
Should I use NvApiWrapper/ADLX or just LHM GPU?

### DISCUSSION THREAD

**Discussion by d-e-n-n-i-s-t** [Rem0o/FanControl.Releases#2680] [Q&A] [answered]
Assuming both LHM and NvApiWrapper/ADLX work and can control the GPU fans in a PC; Which one is preferable? Do NvApiWrapper/ADLX extend the functionality in some way? Is there less overhead using one or the other?

**Comment by Rem0o** [comment, marked as answer]
Both ADLX+NvApiWrapper have way less overhead compared to the LHM built-in GPU support. LHM does a few more routines to get various information, all of which are irrelevant for FanControl.

**Comment by d-e-n-n-i-s-t** [comment]
Thanks man, exactly what I wanted to know! :)
